pom.xml

```diff
@@ -1627,7 +1627,7 @@
         <version.jaxb.api>2.4.0-b180830.0359</version.jaxb.api>
         <com.sun.jaxb.version>2.3.0</com.sun.jaxb.version>
         <com.sun.jaxb.impl.version>2.3.1</com.sun.jaxb.impl.version>
-        <synapse.version>4.0.0-wso2v216</synapse.version>
+        <synapse.version>4.0.0-wso2v217-SNAPSHOT</synapse.version>
```

Comment:
Let's not mention snapshot version. Once the synapse version is released with the fix, we can upgrade it here.

Reply:
Fixed with 238fadfeede47fcf1733dea02bed46be6aefe20e